src/main/java/org/olf/folio/order/OrderImport.java

```diff
@@ -232,15 +233,19 @@ public  JSONArray  upload(String fileName) throws IOException, InterruptedExcept
 				locations.put(location);
 				 
 				
-				//VENDOR REFERENCE NUMBER IF INCLUDED IN THE MARC RECORD:
+				// as of IRIS release vendorDetail is slightly more complex
 				if (vendorItemId != null) {
-					JSONObject vendorDetail = new JSONObject();
-					vendorDetail.put("instructions", "");
-					vendorDetail.put("refNumber", vendorItemId);
-					vendorDetail.put("refNumberType", "Internal vendor number");
-					vendorDetail.put("vendorAccount", "");
-					orderLine.put("vendorDetail", vendorDetail);
-				}
+                    JSONArray referenceNumbers = new JSONArray();
+                    JSONObject vendorDetail = new JSONObject();
+                    vendorDetail.put("instructions", "");
+                    vendorDetail.put("vendorAccount", "");
```

Comment:
I know these were both already here, but no harm in dropping them if we're not setting values, correct?

```suggestion
```